testers/testers/pyta/bin/create_environment.sh

```diff
@@ -7,6 +7,7 @@ create_venv() {
     python${PY_VERSION} -m venv ${VENV_DIR}
     source ${VENV_DIR}/bin/activate
     pip install wheel
+    pip install --upgrade pip
```

Comment:
It's a small thing, but this should probably happen before installing `wheel` (also keeps this consistent with the `py` tester environment creation.